Issue/1325 if statement

sanderr opened the pull request.

sanderr commented:
added recommended scope tests

arnaudsjs commented:
We should add two more things here:

1. Documentation in the Inmanta language reference (see: docs/language.rst)
2. An entry in CHANGELOG.md

bartv commented:
Documentation is also related to #855 
This is one more use case for conditions/expressions

sanderr commented:
Could someone provide me with a useful example for the documentation, or replace the toy example I added? I have added the explanation and the syntax, but I am currently lacking practical application knowledge.

wouterdb commented on docs/language.rst:
```diff
@@ -432,6 +434,25 @@ The syntax is:
+
+.. code-block:: inmanta
+
+    if 1 == 1:
```
```
if nodecount > 1:
  self.cluster_mode = True
else:
  self.cluster_mode = False
end
